rollbar/lib/traverse.py

```diff
@@ -114,24 +116,29 @@ def traverse(obj,
     }
     kw.update(custom_handlers)
 
-    if obj_type is STRING:
-        return string_handler(obj, key=key)
-    elif obj_type is TUPLE:
-        return tuple_handler(tuple(traverse(elem, key=key + (i,), **kw) for i, elem in enumerate(obj)), key=key)
-    elif obj_type is NAMEDTUPLE:
-        return namedtuple_handler(obj._make(traverse(v, key=key + (k,), **kw) for k, v in iteritems(obj._asdict())), key=key)
-    elif obj_type is LIST:
-        return list_handler(list(traverse(elem, key=key + (i,), **kw) for i, elem in enumerate(obj)), key=key)
-    elif obj_type is SET:
-        return set_handler(set(traverse(elem, key=key + (i,), **kw) for i, elem in enumerate(obj)), key=key)
-    elif obj_type is MAPPING:
-        return mapping_handler(dict((k, traverse(v, key=key + (k,), **kw)) for k, v in iteritems(obj)), key=key)
-    elif obj_type is DEFAULT:
-        for handler_type, handler in iteritems(custom_handlers):
-            if isinstance(obj, handler_type):
-                return handler(obj, key=key)
+    try:
+        if obj_type is STRING:
+            return string_handler(obj, key=key)
+        elif obj_type is TUPLE:
+            return tuple_handler(tuple(traverse(elem, key=key + (i,), **kw) for i, elem in enumerate(obj)), key=key)
+        elif obj_type is NAMEDTUPLE:
+            return namedtuple_handler(obj._make(traverse(v, key=key + (k,), **kw) for k, v in iteritems(obj._asdict())), key=key)
+        elif obj_type is LIST:
+            return list_handler(list(traverse(elem, key=key + (i,), **kw) for i, elem in enumerate(obj)), key=key)
+        elif obj_type is SET:
+            return set_handler(set(traverse(elem, key=key + (i,), **kw) for i, elem in enumerate(obj)), key=key)
+        elif obj_type is MAPPING:
+            return mapping_handler(dict((k, traverse(v, key=key + (k,), **kw)) for k, v in iteritems(obj)), key=key)
+        elif obj_type is DEFAULT:
+            for handler_type, handler in iteritems(custom_handlers):
+                if isinstance(obj, handler_type):
+                    return handler(obj, key=key)
+    except:
+        # use the default handler for unknown object types
+        log.exception("Exception while processing Rollbar Item. "
```

Comment:
This error message doesn't seem very informative... if I see this in my logs, I don't know if it's a problem or not, what I should do, etc.